blackbox/docs/admin/system-information.rst

```diff
@@ -1056,25 +1059,33 @@ cluster for each node. Jobs metrics are not persisted across node restarts.
-+------------------------------------+----------------------------------------------------+---------------+
-| ``total_count``                    | Total number of queries executed                   | ``LONG``      |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``mean``                           | The mean query latency in ms                       | ``DOUBLE``    |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``stdev``                          | The standard deviation of the query latencies      | ``DOUBLE``    |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``max``                            | The maximum query latency in ms                    | ``LONG``      |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``min``                            | The minimum query latency in ms                    | ``LONG``      |
-+------------------------------------+----------------------------------------------------+---------------+
-| ``percentiles``                    | An object containing different percentiles         | ``OBJECT``    |
-+------------------------------------+----------------------------------------------------+---------------+
-
++------------------------------+----------------------------------------------------+------------------+
+| Column Name                  | Description                                        |  Return Type     |
++==============================+====================================================+==================+
+| ``node``                     | An object containing the id and name of the node   | ``OBJECT``       |
+|                              | on which the metrics have been sampled.            |                  |
++------------------------------+----------------------------------------------------+------------------+
+| ``classification``           | An object containing the statement classification. | ``OBJECT``       |
++------------------------------+----------------------------------------------------+------------------+
+| ``classification['type']``   | The general type of the statement. Types are:      | ``STRING``       |
+|                              | ``INSERT``, ``SELECT``, ``UPDATE``, ``DELETE``,    |                  |
+|                              | ``COPY``, ``DDL``, and ``MANAGEMENT``.             |                  |
++------------------------------+----------------------------------------------------+------------------+
+| ``classification['labels']`` | The set of operators involved in the statement.    | ``STRING_ARRAY`` |
+|                              | This applies only for ``SELECT`` statements.       |                  |
```

Comment:
We could also include this for insert, delete and update if they involve a query/collect operation. Maybe we can phrase this more openly - by stating that this information isn't provided for all statements.